docs/configuration/alertmanager-integration/grafana-alert-manager.rst

```diff
@@ -1,31 +1,108 @@
 Grafana AlertManager
```

Comment:
You did great work with the screenshots you took. Can you add them here? (If necessary, censor out any keys of course.)